### PR TITLE
Generate kubernetes container definition #11

### PR DESCRIPTION
**Pull request by autumnfound**

### CODE REVIEW

**Comment by autumnfound** [comment]
This needs a quick update to the build, which will be forthcoming

**Review by mbarbero — APPROVED** [review]

File: src/main/k8s/production.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+kind: "Route"
+metadata:
+  name: eclipsefdn-react-membership
+  namespace: foundation-internal-webdev-apps
+  annotations:
+    haproxy.router.openshift.io/timeout: 20s
+    haproxy.router.openshift.io/disable_cookies: "true"
+    haproxy.router.openshift.io/balance: roundrobin
+spec:
+  host: "api.eclipse.org"
```
Comment (chrisguindon) [inline src/main/k8s/production.yml:81]: I don't think we will want to host this on API staging. I've asked @shanda-eclipse  to confirm the URLs on slack but I am thinking we need the following:

membership.eclipse.org (Enrollment form)
membership.eclipse.org/portal (Membership Portal)
Staging:
membership-staging.eclipse.org (Enrollment form)
membership-staging.eclipse.org/portal (Membership Portal)

**Review by chrisguindon — CHANGES_REQUESTED** [review]
Let's focus on getting this on staging before production!

**Comment by shanda-eclipse** [comment]
I can't think of any conflicting uses for membership.eclipse.org, so I think it makes sense to put these on the membership subdomain.

File: src/main/k8s/production.yml (hunk 1 above)
Comment (shanda-eclipse) [reply src/main/k8s/production.yml]: I can't think of any conflicting uses for membership.eclipse.org, so I think it makes sense to put these on the membership subdomain.

**Pull request merged**